__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require('supertest')
+const db = require('../db/connection')
+const app = require('../app/app')
+
+const seed = require('../db/seeds/seed')
+const categoryData = require('../db/data/test-data/categories')
+const commentData = require('../db/data/test-data/comments')
+const reviewData = require('../db/data/test-data/reviews')
+const userData = require('../db/data/test-data/users')
+
+
+beforeEach(() => {
+    return seed({categoryData, commentData, reviewData, userData })
+})
+
+afterAll(() => {
+    db.end()
+})
+
+describe('App', () => {
+    describe('GET /api/categories', () => {
+        it('should when passed an array of objects, return that array of objects', () => {
```

Comment:
It's nice to say what status we are expecting in our test descriptions too! 
Also this isn't the most clear description - what should exactly be returned?